lib/rack/utils.rb

```diff
@@ -7,6 +7,7 @@
 require 'tempfile'
 require 'time'
 require 'cgi/escape'
+require 'erb'
```

Comment:
No need to require `cgi/escape` if `ERB::Escape.html_escape` is available.

Reply:
Good point 👍